shared/base.js

```diff
@@ -29,15 +29,15 @@ function enableOrDisable(tabId, changeInfo, tab) {
   }
 }
 
-function reportIssue(tab, reporterID) {
+function reportIssue(tab, reporterID, tabId) {
   chrome.tabs.captureVisibleTab({ format: "png" }, function(res) {
     let screenshotData = res;
     chrome.tabs.query({ currentWindow: true, active: true }, function(tab) {
       var newTabUrl = `${PREFIX}${encodeURIComponent(
         tab[0].url
       )}&src=${reporterID}`;
       chrome.tabs.create({ url: newTabUrl }, function(tab) {
-        chrome.tabs.executeScript({
+        chrome.tabs.executeScript(tabId, {
```

Comment:
And here we can just use `tab.id`, because the right `tab` is in scope in this callback.